index.html

```diff
@@ -71,9 +71,9 @@ <h2>My Recent Works</h2>
 
             <div class="cards">
                 <div class="card">
-                    <img src="images/Img-Plaholder.png" alt="project pricture">
+                    <img src="images/Img-Plaholder.png" alt="Multi-Post Stories Gain+Glory project image">
```

Comment:
- [ ] For all the alt text, it is always better not to include direct description of the image such "project image". remove these kind of direct descriptions from your alt texts.